manga_auto_translator/main.py

```diff
@@ -23,12 +24,31 @@
     default='MANGA_OCR',
     help=f'OCR strategy to use for character recognition.'
 )
-def cli(input_path, output_path, ocr):
+@click.option(
+    '--translation-api', '-t',
+    type=click.Choice(ALLOWED_TRANSLATION_OPTIONS),
+    default='GOOGLE_TRADUCTION',
+    help='OCR strategy to use for character recognition.'
+)
+@click.option(
+    '--lang-from',
+    type=click.Choice(ALLOWED_TRANSLATION_SOURCE_LANG),
+    default='ja',
+    help='Original language to translate from'
+)
+@click.option(
+    '--lang-to',
+    type=click.Choice(ALLOWED_TRANSLATION_TARGET_LANG),
+    default='fr',
+    help='Language to translate to'
+)
+def cli(input_path, output_path, ocr, translation_api, lang_from, lang_to):
     scans = ScanIOManager.load_scans(input_path)
 
     ocr_strategy = OcrStrategyFactory(strategy=ocr).create()
+    translation_strategy = TraductionStrategyFactory(strategy=translation_api).create()
```

Comment:
This is great and I'm doing the same thing.
The way you call it is fine, but can we find a more elegant way to call `create()` instead of instantiating then directly call it ? Can we consider the option to create static method and put `translation_api` as argument for `create()` method ?

Reply:
I agree with you. I think this would be better. 
I was thinking of making a follow up PR to suggest this because it's also implemented like that for the OCR.